deepfence_frontend/apps/dashboard/src/features/settings/components/SuccessModal.tsx

```diff
@@ -0,0 +1,24 @@
+import { IconContext } from 'react-icons';
+import { HiOutlineCheckCircle } from 'react-icons/hi';
```

Comment:
@harishshg We do not need a new Success Modal. Idea is to not close the Modal automatically.
1. If submit is success, display the success message on the same modal where you submit the form.
2. If submit is failure, display the failure message on the same modal which where you submit the form.
In both the cases, let user close the modal and we just display the message. No need to close previous modal and open a new success modal

Reply:
I think this is fine, `SuccessModal` is not a modal here, but content. So it makes sense if you rename it to `SuccessModalContent`. I am not sure if we need error modal, since errors will be shown in the form modal itself.

Reply:
Renamed the SuccessModal to SuccessModalContent